scala_proto/scala_proto.bzl

```diff
@@ -414,7 +414,7 @@ Example:
 
 Args:
     name: A unique name for this rule
-    deps: Proto library targets that this rule depends on (must be of type proto_library)
+    deps: Proto library targets or jvm targets that this rule depends on
```

Comment:
this can't be any jvm target, right? You are expecting java_proto_library?

Reply:
Well, any jvm targets that you pass here are going to be passed through as deps to the final output scala_library. Does that seem unreasonable?